src/States/Order/OrderState.php

```diff
@@ -85,9 +84,4 @@ public static function getFrontendFormatter(...$args): string|array
                 ->map(fn ($key) => (new $key(''))->color()),
         ];
     }
-
-    public function toArray(): array|string
```

Comment:
does removing this have any effect on the existing code-base. Or was this just added but never used?

Reply:
the to array is in the flux state base class, as well as the arrayable contract